log/logger.go

```diff
@@ -9,6 +15,59 @@ type Logger interface {
 	Error() Logger
 	Fatal() Logger
 
-	Logf(format string, a ...interface{})
-	LogErrorf(format string, a ...interface{}) error
+	Log(message string)
+	Logf(format string, args ...interface{})
```

Comment:
We could use `Logf(format string, args ...interface{})` for cases where we'd want to pass in a string without args, but adding `Log(msg string)` avoids any subtle bugs where the caller meant to pass in an interpolated string but forgot to add in the args.